src/main/java/com/teamabnormals/abnormals_core/core/examples/ExampleTileEntityRegistry.java

```diff
@@ -3,11 +3,13 @@
 import com.teamabnormals.abnormals_core.common.blocks.AbnormalsBeehiveBlock;
 import com.teamabnormals.abnormals_core.common.blocks.chest.AbnormalsChestBlock;
 import com.teamabnormals.abnormals_core.common.blocks.chest.AbnormalsTrappedChestBlock;
-import com.teamabnormals.abnormals_core.common.blocks.sign.AbnormalsAbstractSignBlock;
-import com.teamabnormals.abnormals_core.common.tileentity.*;
+import com.teamabnormals.abnormals_core.common.blocks.sign.ITexturedSign;
+import com.teamabnormals.abnormals_core.common.tileentity.AbnormalsBeehiveTileEntity;
+import com.teamabnormals.abnormals_core.common.tileentity.AbnormalsChestTileEntity;
+import com.teamabnormals.abnormals_core.common.tileentity.AbnormalsSignTileEntity;
+import com.teamabnormals.abnormals_core.common.tileentity.AbnormalsTrappedChestTileEntity;
```

Comment:
Should be importing the tile entities like so, `import com.teamabnormals.abnormals_core.common.tileentity.*;`